config/initializers/devise.rb

```diff
@@ -180,15 +180,12 @@
   # :time  = Re-enables login after a certain amount of time (see :unlock_in below)
   # :both  = Enables both strategies
   # :none  = No unlock strategy. You should handle unlocking by yourself.
-  config.unlock_strategy = :both
+  config.unlock_strategy = :email
```

Comment:
not sure if this should be `:none` or `:email`

Reply:
I think email is fine as we do want to allow users to unlock themselves if its triggered for schools that are active.